daft/expressions/expressions.py

```diff
@@ -1137,7 +1410,22 @@ def endswith(self, suffix: str | Expression) -> Expression:
         """Checks whether each string ends with the given pattern in a string column
 
         Example:
-            >>> col("x").str.endswith(col("foo"))
+            >>> import daft
+            >>> df = daft.from_pydict({"x": ["foo", "bar", "baz"]})
```

Comment:
Let's have an example that isn't the string itself but ends with the string, i.e. "foobar" and endswith "bar"